src/spryker/docker/image/nginx/root/etc/nginx/conf.d/default.conf.twig

```diff
@@ -0,0 +1,96 @@
+server {
+    listen 80 default_server;
+    listen 443 ssl http2 default_server;
+
+    server_name {{ @('yves-host') }};
+
+    include snippets/certificate.conf;
+    include snippets/ssl-params.conf;
+
+    root /app/public/Yves;
+
+    # Static files - allow only specified here paths
+    # all other resources should be served via static host (and cached, if possible, via reverse proxy or cdn)
+    location ~ (/assets/|/maintenance.html|/favicon.ico|/crossdomain.xml) {
+        access_log        off;
+        expires           30d;
+        add_header Pragma public;
+        add_header Cache-Control "public";
+        try_files $uri $uri/ =404;
+        #more_clear_headers 'X-Powered-By' 'X-Store' 'X-Locale' 'X-Env' 'Server';
+    }
+
+    location ~ \.(jpg|gif|png|css|js|html|xml|ico|txt|csv|map)$ {
+        access_log off;
+        expires 30d;
+        add_header Pragma public;
+        add_header Cache-Control "public";
+        try_files $uri /index.php?$args;
+        #more_clear_headers 'X-Powered-By' 'X-Store' 'X-Locale' 'X-Env' 'Server';
+    }
+
+    # PHP application
+    location / {
+        ## CORS - Allow ajax requests from http to https webservices on the same domain
+        #more_set_headers "Access-Control-Allow-Origin: http://$server_name";
+        #more_set_headers "Access-Control-Allow-Credentials: true";
+        #more_set_headers "Access-Control-Allow-Headers: Authorization";
+
+        # Frontend - force browser to use new rendering engine
+        #more_set_headers "X-UA-Compatible: IE=Edge,chrome=1";
+
+        # Terminate OPTIONS requests immediately. No need for calling php
+        # OPTIONS is used by Ajax from http to https as a pre-flight-request
+        # see http://en.wikipedia.org/wiki/Cross-origin_resource_sharing
+        if ($request_method = OPTIONS) {
+            return 200;
+        }
+
+        add_header X-Server $hostname;
+        fastcgi_pass php-fpm:9000;
```

Comment:
This passes every request apart from the static assets listed above to PHP, are there any access routes that should be denied or not allow PHP processing (e.g. upload directories)?

Reply:
There is an `assets` directory, but that is handled here https://github.com/inviqa/harness-base-php/blob/aafebc1880e0dc1f6bfb68407a29f50c6b1e847c/src/spryker/docker/image/nginx/root/etc/nginx/conf.d/default.conf.twig#L14